src/Query/index.ts

```diff
@@ -181,6 +181,20 @@ export class Query {
       },
     });
 
+    parts.push({
+      text: {
+        query: terms,
+        path: [
```

Comment:
this works for now but i think ideally we would get this programmatically from SearchIndex.trieFacets

Reply:
going to drop in a ticket for this because it's taking longer than expected 😞